samples/CommunityToolkit.Maui.Sample/MauiProgram.cs

```diff
@@ -51,12 +50,12 @@ public static MauiApp CreateMauiApp()
 	{
 		var builder = MauiApp.CreateBuilder()
 #if DEBUG
-								.UseMauiCommunityToolkit(options =>
+								.UseMauiCommunityToolkit(static options =>
```

Comment:
Hi @brminnick! Thanks for looking into this! Can you expand the use of `static` here, please? First time seeing this in C#. I tried to search for a doc related to this feature but without a luck. Thanks in advance!